src/Footer.js

```diff
@@ -0,0 +1,12 @@
+import {Component} from "react";
+
+export default class Footer extends Component{
+    render(){
+        return(
+            <footer>
+                <h4>Created By: Samuel Panek</h4>
```

Comment:
Hi there, good job again on Lab * 👍 

I am leaving a comment here because on deployed site, your footer is sitting directly over your `Explore` Button. I would take a look at footer/styles when you can.